packages/web-react/src/components/Header/README.md

```diff
@@ -4,398 +4,541 @@ The Header is a highly variable and customizable component. It comes in several
-  </HeaderDialogActions>
-</HeaderDialog>
-// HeaderDialog #2
-<HeaderDialog
-  id="header_dialog_example_2"
-  aria-label="User menu"
-  isOpen={isUserMenuOpen}
-  onClose={handleUserMenuClose}
->
-  <HeaderDialogCloseButton />
-  <HeaderDialogActions color="primary" aria-label="User menu">
-    <HeaderDialogNav>
-      <HeaderDialogNavItem>
-        <HeaderDialogLink href="…">Dashboard</HeaderDialogLink>
-      </HeaderDialogNavItem>
-      <HeaderDialogNavItem>
-        <HeaderDialogLink href="…">Profile</HeaderDialogLink>
-      </HeaderDialogNavItem>
-      …
-      <HeaderDialogNavItem>
-        <HeaderDialogButton>Sign out</HeaderDialogButton>
-      </HeaderDialogNavItem>
-    </HeaderDialogNav>
-  </HeaderDialogActions>
-</HeaderDialog>
 ```
 
-## Header
+## API
 
-### API
+| Name               | Type                          | Default       | Required | Description                         |
+| ------------------ | ----------------------------- | ------------- | -------- | ----------------------------------- |
+| `children`         | `ReactNode`                   | —             | ✕        | Children node                       |
+| `color`            | [`transparent` \| `inverted`] | `transparent` | ✕        | Color variant                       |
+| `isSimple`         | `bool`                        | `false`       | ✕        | Shorter, centered version of Header |
+| `UNSAFE_className` | `string`                      | —             | ✕        | Custom class name                   |
+| `UNSAFE_style`     | `CSSProperties`               | —             | ✕        | Custom style                        |
 
-| Name               | Type                          | Default | Required | Description                |
-| ------------------ | ----------------------------- | ------- | -------- | -------------------------- |
-| `children`         | `ReactNode`                   | —       | ✕        | Children node              |
-| `color`            | [`inverted` \| `transparent`] | —       | ✕        | Header background color    |
-| `isSimple`         | `bool`                        | —       | ✕        | If header should be simple |
-| `UNSAFE_className` | `string`                      | —       | ✕        | Header custom class name   |
-| `UNSAFE_style`     | `CSSProperties`               | —       | ✕        | Header custom style        |
+The component implements the [`HTMLElement`][mdn-api-html-element] interface.
 
-The rest of the properties are based on the type of `HTMLElement`. [Docs][HTMLElementDocs]
+## Supported Content
 
-## HeaderButton
+To create a responsive header with top-level navigation, there are the following
+building blocks to use:
 
-### API
+1. Inside Header:
+1. mobile-only actions, including toggle button by default,
+1. desktop-only actions with primary and secondary action slots.
+1. Inside Header Dialog:
+1. primary actions slot (all breakpoints),
+1. secondary actions slot (all breakpoints).
```

Comment:
I see only 1.s here 🤔 
<img width="145" alt="image" src="https://github.com/lmc-eu/spirit-design-system/assets/7041807/8094919b-292a-4054-ac19-4aa6fd12e646">

Reply:
But in the rendered Readme it is ok. 🤷

Reply:
Thanks for pointing out, the list got flat during copy-pasting.